dlt/pipeline/pipeline.py

```diff
@@ -555,7 +555,9 @@ def load(
             with signals.delayed_signals():
                 runner.run_pool(load_step.config, load_step)
             info: LoadInfo = self._get_step_info(load_step)
+
             self.first_run = False
+            load_step.maybe_trancate_staging_dataset(self.default_schema)
```

Comment:
makes sense to put it into a separate method, but I'd prefer to have `load.py` self-contained so please add it at the end of `complete_package` method

Reply:
There were failures, if I do truncation somewhere inside the `Load`. Let's see...